.github/workflows/snyk.yaml

```diff
@@ -23,9 +23,9 @@ jobs:
       id: scan
       uses: puppetlabs/security-snyk-clojure-action@v2
       with:
-        snykToken: ${{ secrets.SNYK_GENERAL_KEY }}
+        snykToken: ${{ secrets.SNYK_PE_KEY }}
```

Comment:
whoops, I should've looked at the secrets before I merged.. looks like the secret is called `SNYK_PE_TOKEN`, not `SNYK_PE_KEY`.

Reply:
fix here: https://github.com/puppetlabs/clj-kitchensink/pull/117